lib/src/components/NxList/NxListDescription.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {NxListDescriptionProps, nxListDescriptionTermPropTypes} from './types';
+
+const NxListDescription = (props: NxListDescriptionProps) => {
```

Comment:
Better as convenience component

Reply:
> Better as convenience component

Would like more clarification on this. Should we revert it back to a convenience component or add className and extra attrs?

Reply:
> Would like more clarification on this. I have added support for refs, className and extra attrs. Or are you suggesting we revert it back to a convenience component?

Aside from the support for `refs`, which isn't crucial, the `withClass` function used to construct convenience components already does all that.  So I'm saying use that function to define this component rather than setting it all up manually.

Reply:
> > Would like more clarification on this. I have added support for refs, className and extra attrs. Or are you suggesting we revert it back to a convenience component?
> 
> Aside from the support for `refs`, which isn't crucial, the `withClass` function used to construct convenience components already does all that. So I'm saying use that function to define this component rather than setting it all up manually.

Used withClass as suggested.
b74047c3d07f855bc844e3f6770b79800310b377